src/org/janelia/mipav/plugins/worm/untwisting/SelectionChartPanel.java

```diff
@@ -0,0 +1,229 @@
+package org.janelia.mipav.plugins.worm.untwisting;
+
+import org.jfree.chart.*;
+import org.jfree.chart.axis.ValueAxis;
+import org.jfree.chart.event.MarkerChangeEvent;
+import org.jfree.chart.event.MarkerChangeListener;
+import org.jfree.chart.plot.PlotOrientation;
+import org.jfree.chart.plot.ValueMarker;
+import org.jfree.chart.plot.XYPlot;
+import org.jfree.chart.renderer.xy.XYLineAndShapeRenderer;
+import org.jfree.chart.ui.RectangleAnchor;
+import org.jfree.chart.ui.TextAnchor;
+import org.jfree.chart.ui.Layer;
+import org.jfree.data.xy.XYSeries;
+import org.jfree.data.xy.XYSeriesCollection;
+
+import WildMagic.LibFoundation.Mathematics.Vector3f;
+
+import java.awt.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.awt.geom.Rectangle2D;
+import java.util.List;
+
+public class SelectionChartPanel extends ChartPanel implements MarkerChangeListener {
+
+
+/**
+ * CustomChartPanel is a custom extension of the ChartPanel class from JFreeChart
+ * used to display and interact with a dynamic XY line chart. It includes custom
+ * behaviors for handling marker changes and mouse drag events.
+ */
+	private static final long serialVersionUID = 1L;
+	private JFreeChart selectionChart;
+	private List<Vector3f> chart3DPoints;
+	private PlugInDialogVolumeRenderDualJanelia parent;
+
+	public SelectionChartPanel(List<Float> values, String title, PlugInDialogVolumeRenderDualJanelia parent) {
+		super(null);
+		this.selectionChart = createChart(values, title, this);
+		this.parent = parent;
+		this.setChart(selectionChart);
+		setPreferredSize(new Dimension(100,100));
```

Comment:
We probably do not need this anymore.

```suggestion
```

Reply:
- [ ] added //comment for if remove this line, the plot get larger vertically and horizontally